scripts/simulator/simulate.py

```
@@ -97,16 +97,16 @@ def __init__(self, user_persona_file: str, personality: str = "companionship", *
-        """Load persona content from markdown file in sim_personas directory."""
+    def _load_character(self, character_filename: str) -> str:
+        """Load character content from markdown file in sim_characters directory."""
         # Get the directory of the current script
         script_dir = Path(__file__).parent
-        persona_path = script_dir / "sim_personas" / f"{persona_filename}.md"
+        character_path = script_dir / "sim_characters" / f"{character_filename}.md"
 
-        if not persona_path.exists():
-            raise FileNotFoundError(f"Persona file not found: {persona_path}")  # noqa: TRY003
+        if not character_path.exists():
+            raise FileNotFoundError(f"Character file not found: {character_path}")  # noqa: TRY003
```

Ruff (0.13.1)

Unused `noqa` directive (unused: `TRY003`)

Remove unused `noqa` directive

(RUF100)